Stack/Opc.Ua.Core/Schema/SecuredApplicationHelpers.cs

```diff
@@ -388,7 +388,12 @@ public static byte CalculateSecurityLevel(MessageSecurityMode mode, string polic
             byte result = 0;
             switch (policyUri)
             {
-                case SecurityPolicies.Basic128Rsa15: result = 2; break;
+                case SecurityPolicies.Basic128Rsa15:
+                {
+                    Utils.LogWarning("Security level Basic 128 Rsa 15 requested as Security Policy {policy} - Not Recommended.", policyUri);
```

Comment:
I would change this warning to "Deprecated Security Policy Basic128Rsa15 requested - Not rcommended.